src/ros_filter_utilities.cpp

```diff
@@ -98,14 +98,15 @@ namespace RosFilterUtilities
                            const std::string &targetFrame,
                            const std::string &sourceFrame,
                            const ros::Time &time,
+                           const ros::Duration &timeout,
```

Comment:
Hi Tom, took some time but finally got to it. Added an additional method to maintain the old signature.